core/ecschema-metadata/src/Deserialization/JsonProps.ts

```diff
@@ -267,6 +267,16 @@ export interface ConstantProps extends SchemaItemProps {
  */
 export type SchemaItemFormatProps = SchemaItemProps & FormatProps;
 
+/**
+ * This interface defines properties necessary to support persistence of a set of formats.
+ * @beta
+ */
+export interface FormatSet {
+  name: string;
+  label: string;
+  formats: { [kindOfQuantityId: string]: SchemaItemFormatProps }
+}
```

Comment:
Assuming we'll have a number of these FormatSets coming from different sources, should we have some "priority" attribute to know which ones takes precedence?

I believe we'd have the following order: user format sets > imodel format sets > itwin format sets > organization format sets > app default format sets? > koqs in schema.

Reply:
sounds good, just wanted to make sure this is thought through